grammar.js

```diff
@@ -276,6 +276,12 @@ module.exports = grammar({
       $.implicit_object_call_chainable,
       $._implicit_object_call,
     ],
+
+    // Ensure parens marking pseudo method args bind to them
+    [
+      $.pseudo_call_argument_list,
+      $._parenthesized_type,
+    ],
```

Comment:
I had an old branch I had started for pseudo-methods, here's how I avoided conflicts:

``` javascript
precedences: [
  // Ensure `is_a? Foo . class` parses as `is_a?(Foo.class)`, not `is_a?(Foo).class`
  [
    'atomic_type',
    $.is_a,
  ],
]

// ...
    is_a: $ => {
      const receiver = field('receiver', $._expression)
      const method = field('method', alias('is_a?', $.identifier))
      // We don't need to worry about parentheses here, because
      // _parenthesized_type will handle them
      const args = field('arguments', alias($._type, $.arguments_list))

      return seq(
        receiver,
        '.',
        method,
        args,
      )
    },
```

Reply:
Wait, I just realized this doesn't work in some situations...

Reply:
Ignore my comments above, what you have here looks correct